cni-plugin/pkg/install/install.go

```diff
@@ -141,7 +140,7 @@ func Install() error {
 	c.ServiceAccountToken = make([]byte, 0)
 	var err error
 	if fileExists(serviceAccountTokenFile) {
-		log.Info("Running as a Kubernetes pod")
+		logrus.Info("Running as a Kubernetes pod")
```

Comment:
We use `logrus` in all our components as well as this file. `prometheus/common/log` is removed from the official package a while back.